runtime/rococo/src/propose_parachain.rs

```diff
@@ -289,6 +300,19 @@ decl_module! {
 
 			pallet_balances::Module::<T>::unreserve(&info.proposer, T::ProposeDeposit::get());
 		}
+
+		/// Add a new validator to the set.
+		#[weight = 100_000]
+		fn register_validator(
```

Comment:
this should be `register_validators` (plural)
```suggestion
		/// Add new validators to the set.
		#[weight = 100_000]
		fn register_validators(
```